### PR TITLE
fix: Update default database to PostgreSQL in v4

### LINKED ISSUE #4980: Update default database to PostgreSQL in Common Config in V4
# 🚀 Feature Request

### Relevant Package [**REQUIRED**]

 This feature request is for all EdgeX service which uses database, config provider and registry services.

### Description [**REQUIRED**]
 The following changes will be applied starting from V4:
- The default database is changed from redis to PostgreSQL in Common Config.
- The default Message Bus is changed from redis to mqtt in Common Config.
- The default configuration provider and registry host are changed from Consul to Core Keeper.
- Remove support-scheduler in the code.

### Describe the solution you'd like
 Related configuration and Dockerfile will be updated.

### PR DESCRIPTION
**Pull request by lindseysimple**
Resolves #4980. Update default database, msg bus, config provider and registry host. Remove support-scheduler code.

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: cmd/support-notifications/res/configuration.yaml [hunk 1]
```diff
@@ -36,10 +31,6 @@ MessageBus:
 
 Database:
   Name: notifications
```
Comment (cloudxxx8) [inline cmd/support-notifications/res/configuration.yaml:33]: do we still need the database name?

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]
modify the configuration file of support-cron-scheduler to use the commom config database setting.
it doesn't need to use its own anymore

File: cmd/support-notifications/res/configuration.yaml (hunk 1 above)
Comment (lindseysimple) [reply cmd/support-notifications/res/configuration.yaml]: It's not needed by Postgres. Removed.

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**